Bindings.xml

```diff
@@ -0,0 +1,5 @@
+<Bindings>
+    <Binding name="WEEKLY REWARDS" category="WeeklyRewards">
+        TOGGLE_WEEKLY_REWARDS_WINDOW()
```

Comment:
It would be better if you could have the following changes:  
1. Expand the `Main:ToggleWindow()` here, then we don't need to introduce the new global var `TOGGLE_WEEKLY_REWARDS_WINDOW`
2. Replace `name="WEEKLY REWARDS"` to `name="WeeklyRewards"`, then you can reuse the predefined `_G["BINDING_NAME_WeeklyRewards"]` and the keybinding UI will show: **"Show/Hide the window"**

Reply:
here you go:

```xml
<Bindings>
    <Binding name="WeeklyRewards" category="WeeklyRewards">
        WeeklyRewards:ExecuteChatCommands()
    </Binding>
</Bindings>
```